lte/gateway/python/integ_tests/federated_tests/configs/hss.yml

```diff
@@ -0,0 +1,27 @@
+---
+#
+# Copyright 2020 The Magma Authors.
+
+# This source code is licensed under the BSD-style license found in the
+# LICENSE file in the root directory of this source tree.
+
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# HSS Config
+#
+# ---
+#subscribers:
+#  <imsi>:
+#    <auth_key>: - required (hex string)
+#    <non_3gpp_enabled>: - optional (bool)
+subscribers:
+  "001010000000001":
+    auth_key: "000102030405060708090A0B0C0D0E0F"
+    lte_auth_next_seq: 1
+  "001010000000002":
+    auth_key: "000102030405060708090A0B0C0D0E0F"
+    lte_auth_next_seq: 1
```

Comment:
here we add some test subscribers

Maybe we will place them on some grcp calls in the future instead of having them on a config file